src/block.rs

```diff
@@ -5,9 +5,9 @@ use std::str::FromStr;
 
 pub mod validator;
 
-use crate::errorkey::ErrorKey;
-use crate::errors::{error, error_info};
 use crate::parse::pdxfile::{parse_pdx_macro, LocalMacros};
+
```

Comment:
These extra blank lines will be a problem because then `cargo fmt` won't nicely alphabetize the imports anymore :)